packages/itwin/grouping-mapping-widget/src/widget/components/HorizontalTile.tsx

```diff
@@ -20,11 +20,11 @@ export const HorizontalTile = ({ title, subText, onClickTitle, titleTooltip, sub
 
   return (
     <div className="gmw-horizontal-tile-container" data-testid="gmw-horizontal-tile">
-      <div className="body">
+      <div className="gmw-body">
         <Text className={`body-text ${onClickTitle ? "iui-anchor" : ""}`} onClick={onClickTitle} variant="body" title={titleTooltip}>{title}</Text>
```

Comment:
`body-text` class is missing a prefix. Same applies for the `Text` element below.

Reply:
Done